appveyor.yml

```diff
@@ -2,7 +2,7 @@
 version: "{build}"
 
 cache:
-  - '%APPDATA%\npm-cache'
+  - '%APPDATA%\npm-cache -> package.json'
```

Comment:
Here's how I do it. https://github.com/kellyselden/ember-i18n-csv/blob/master/appveyor.yml#L36 and https://github.com/kellyselden/ember-i18n-csv/blob/master/.appveyorcachesucks
Sometimes I needed to kick the cache (without updating package.json). Not telling you to change your commit, just thought I'd share another way. 😄